README.md

```diff
@@ -61,7 +61,7 @@ After you're finished please remove all the comments and instructions!
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [OPTIONAL] I think it's better to remove the guidelines. Those lines started with `>` are guides to creating your README file and don't need to be part of the content. Your README will look more professional.